src/sdb.h

```diff
@@ -206,6 +206,12 @@ SDB_API bool sdb_disk_insert(Sdb* s, const char *key, const char *val);
 SDB_API bool sdb_disk_finish(Sdb* s);
 SDB_API bool sdb_disk_unlink(Sdb* s);
 
+/* plaintext sdb files */
+SDB_API bool sdb_text_save_fd(Sdb *s, int fd, bool sort);
```

Comment:
there's still an fd here

Reply:
That is correct, because the saving directly streams out the data without having to accumulate it in memory first. How would you solve it instead?

Reply:
but this should be only an internal function, not exposed publickly. windows shouldnt be using this integer file descriptors things. and passing the filename should be enough imho.